### ISSUE TITLE
Necrotic revival surgery "Bionecrosize" surgery step not functioning

### ISSUE THREAD

**Issue by MadlyEvilWaffle** [tgstation/tgstation#67825]
## Reproduction:

The necrotic revival surgery doesnt seem to be working correctly; Surgery steps work fine until you reach "bionecrosize". At which point it doesnt work. Tried the surgery step many times, both with a syringe of zombie powder for the last step + having some injected and then trying again after. I have a feeling that this has something to do with the chemical purity PR because I remember having tried this before that and it worked and its the only thing i can think of that has changed in relevance to how the reagent syringe is used. 

Tried having a look at the code, from my very limited understanding im pretty sure that it is infact broken and not me misunderstanding the step.

**Comment by RandomGamer123** [comment]
Can't reproduce, are you sure there were 5 pure units of zombie powder in the bloodstream (if you injected 5, fermichem means that there may have been under 5)

**Comment by the-orange-cow** [comment]
Round ID: 189374
BYOND Version: 514.1583
Server revision compiled on: 
Master commit: [b24e0291479fc292ef89efe29708340e42f1e71f](https://www.github.com/tgstation/tgstation/commit/b24e0291479fc292ef89efe29708340e42f1e71f)
TGS version: /tg/station 13 Server v3.2.6.2
DMAPI version: 3.2.1.3

I just tried it.  Got to the bionecrosize step, had a syringe full of rezadone.  Clicked, and it gave me blue text saying I was trying to make a romerol tumor.  However, the cast bar completed I didn;t get the text indicating it was successful, just nothing, and it was still on that bionecrotize step.  The syrynge was full of rezadone, and I tried doing it while it was set to inject 5u and 15u.  My target was alive, if that makes a difference.
